test/backup.ts

```diff
@@ -223,6 +224,38 @@ describe('Bigtable/Backup', () => {
     });
   });
 
+  describe('copy', () => {
+    it.only('should correctly copy backup from the Cluster', done => {
+      const backupId = generateId('backup');
+      const newBackupId = generateId('backup');
+      const oldBackup = new Backup(CLUSTER, backupId);
```

Comment:
nit - for clarity, I would rename these `backup` and `copiedBackup`

Reply:
Done